docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -83,6 +91,36 @@ configurations.
 |==================================================================================================================================================================================
 
 
+[discrete]
+[[elser-qualitative-benchmarks]]
+=== Qualitative benchmarks
+
+The metric that is used to evaluate ELSER's ranking ability is the Normalized 
+Discounted Cumulative Gain (NDCG), which is the most complete metric as it can 
```

Comment:
I would replace "the most complete metric" by "an interesting metric". There might be some obscure metric capturing something similar than NDCG and you could argue than NDCG is only looking at relevancy which is not a complete picture of information retrieval (you could add performance, index size, ...). But I'm being picky maybe !

Reply:
Thank you for the suggestion and the explanation, @qherreros. I addressed in [c29b337](https://github.com/elastic/stack-docs/pull/2472/commits/c29b3371266e06e6d03c35f668af57bc3853ef17). Eventually, I removed the adjective from the sentence and didn't replace it with another one. Let me know if you'd like to add or remove anything!